### PR TITLE
Navbar fix for FireFox

### PR DESCRIPTION
**Pull request by Phreshhh**
## Brief description

Navbar fix for Firefox.

### CODE REVIEW

File: src/components/_navbar.scss [hunk 1]
```diff
@@ -107,6 +107,13 @@ nav {
     }
   }
 
+  .collapsible {
+    @include resp(small) {
+      width: 100%;
+      margin-top: 50px;
```
Comment (TotomInc) [inline src/components/_navbar.scss]: For the tests to pass, you need to move-up the `margin-top` property before the `width: 100%`.

Comment (Phreshhh) [reply src/components/_navbar.scss]: OMG..  give me a sec. 😈

Comment (TotomInc) [reply src/components/_navbar.scss]: You can see what is wrong with the tests by clicking on the red-cross and then clicking on *Travis Ci details*.

You will see in the console output a detailed error thrown by Stylelint:

image

**Comment by Phreshhh** [comment]
Finally!

Sorry about this, but the stylelint test not show error to me..

Thanks Totomlnc! Best regards.

**Comment by TotomInc** [comment]
@Phreshhh there is a small problem, the navbar seems to take too much height when not opened. This is happening on Chrome, Firefox and Safari, other than that everything works well.

**Before:**

capture d ecran 2019-01-29 a 15 17 53

**After:**

capture d ecran 2019-01-29 a 15 17 48

**Comment by TotomInc** [comment]
@Phreshhh next step is to add back the border on the burger button.

**Comment by TotomInc** [comment]
Seems good for me! Waiting for @rhyneav review, don't worry this review can take some days. It will be merged by rhyneav. Thanks again for contributing to PaperCSS.

**Comment by Phreshhh** [comment]
All feature bugs cleared. Thanks me too for the framework ;)

**Comment by rhyneav** [comment]
Hey @Phreshhh thank you for the PR! What exactly was this fixing in Firefox? 

I also saw that you had changed the markup from a `button` to a `label`. This is fine since it fixes the bug, but I do think that we should keep the `button` selector in the LESS as well. This way it is still backwards compatible with anyone using the navigation with a`button`. What do you think?

**Comment by Phreshhh** [comment]
Hi @rhyneav !

I thinked about it when i changed, but i did't see other solution for fix it then change the markup. :(

I think either-or.

**Comment by rhyneav** [comment]
I think you can leave both with maybe a comment about why both are there. It's not the prettiest thing, but is safe until we launch the next major version.

```SCSS
.collapsible > button, // Leaving for backwards compatibility. See docs for proper usage
.collapsible > label {
  // those pretty styles
}
```

**Comment by Phreshhh** [comment]
I put back the button styles, now works with old layout also same as before.
The old (with button) is not works FireFox but works in Edge and Chrome and the new (without button) works in all browser.

**Comment by rhyneav** [comment]
Thank you for updating!

**Pull request merged**